specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2021-01-01/bms.json

```diff
@@ -0,0 +1,10157 @@
+          "type": "boolean"
+        },
+        "subinquireditemcount": {
+          "format": "int32",
+          "description": "For instance or AG, indicates number of DB's present",
+          "type": "integer"
+        },
+        "subWorkloadItemCount": {
+          "format": "int32",
+          "description": "For instance or AG, indicates number of DB's to be protected",
+          "type": "integer"
+        }
+      },
+      "x-ms-discriminator-value": "AzureVmWorkloadItem"
+    },
+    "AzureVmWorkloadProtectableItem": {
+      "description": "Azure VM workload-specific protectable item.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/WorkloadProtectableItem"
+        }
+      ],
+      "properties": {
+        "parentName": {
+          "description": "Name for instance or AG",
+          "type": "string"
+        },
+        "parentUniqueName": {
+          "description": "Parent Unique Name is added to provide the service formatted URI Name of the Parent\r\nOnly Applicable for data bases where the parent would be either Instance or a SQL AG.",
+          "type": "string"
+        },
+        "serverName": {
+          "description": "Host/Cluster Name for instance or AG",
+          "type": "string"
+        },
+        "isAutoProtectable": {
+          "description": "Indicates if protectable item is auto-protectable",
+          "type": "boolean"
+        },
+        "isAutoProtected": {
+          "description": "Indicates if protectable item is auto-protected",
+          "type": "boolean"
+        },
+        "subinquireditemcount": {
+          "format": "int32",
+          "description": "For instance or AG, indicates number of DB's present",
+          "type": "integer"
+        },
+        "subprotectableitemcount": {
+          "format": "int32",
+          "description": "For instance or AG, indicates number of DB's to be protected",
+          "type": "integer"
+        },
+        "prebackupvalidation": {
+          "$ref": "#/definitions/PreBackupValidation",
+          "description": "Pre-backup validation for protectable objects"
+        }
+      },
+      "x-ms-discriminator-value": "AzureVmWorkloadProtectableItem"
+    },
+    "AzureVmWorkloadSAPAseDatabaseWorkloadItem": {
+      "description": "Azure VM workload-specific workload item representing SAP ASE Database.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AzureVmWorkloadItem"
+        }
+      ],
+      "x-ms-discriminator-value": "SAPAseDatabase"
+    },
+    "AzureVmWorkloadSAPAseSystemProtectableItem": {
+      "description": "Azure VM workload-specific protectable item representing SAP ASE System.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AzureVmWorkloadProtectableItem"
+        }
+      ],
+      "x-ms-discriminator-value": "SAPAseSystem"
+    },
+    "AzureVmWorkloadSAPAseSystemWorkloadItem": {
+      "descriptio
```

Comment:
MoveRPAcrossTiersRequestResource is the body of a POST request.  It does not make sense to have top level resource properties like "id", "name", "location", etc...

Reply:
Changed the request body to MoveRPAcrossTiersRequest and removed MoveRPAcrossTiersRequestResource